util/topsql/stmtstats/kvexeccount.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stmtstats
+
+import (
+	"sync"
+
+	"github.com/tikv/client-go/v2/tikvrpc"
+	"github.com/tikv/client-go/v2/tikvrpc/interceptor"
+)
+
+// CreateKvExecCounter creates an associated KvExecCounter from StatementStats.
+// The created KvExecCounter can only be used during a single statement execution
+// and cannot be reused.
+func (s *StatementStats) CreateKvExecCounter(sqlDigest, planDigest []byte) *KvExecCounter {
```

Comment:
It may not be so pure for `KvExecCounter`, because we will directly face `KvExecCounter` in `executor`, so it is necessary to keep `CreateKvExecCounter` public.